src/styles/cssVariables/cssVariables.ts

```diff
@@ -7,7 +7,7 @@ export enum CssVariables {
   lightBlue = "var(--lightBlue)",
```

Comment:
in ref of `src/styles/cssVariables/cssVariables.css` the file should be just call `variables` the parent folder already have `css` in its name, no need to be redundant.